vizro-core/src/vizro/models/_components/table.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from typing import List, Literal
+
+from dash import html
+from pydantic import Field, PrivateAttr, validator
+
+from vizro.managers import data_manager
+from vizro.models import Action, VizroBaseModel
+from vizro.models._action._actions_chain import _action_validator_factory
+from vizro.models._models_utils import _log_call
+from vizro.models.types import CapturedCallable
+
+logger = logging.getLogger(__name__)
+
+
+class Table(VizroBaseModel):
+    """Wrapper for table components to visualize in dashboard.
+
+    Args:
+        type (Literal["table"]): Defaults to `"table"`.
+        table (CapturedCallable): Table like object to be displayed. Current choices include:
+            [`dash_table.DataTable`](https://dash.plotly.com/datatable).
+        actions (List[Action]): See [`Action`][vizro.models.Action]. Defaults to `[]`.
+    """
+
+    type: Literal["table"] = "table"
+    table: CapturedCallable = Field(..., description="Table to be visualized on dashboard")
```

Comment:
This field will need an `import_path` when we've worked out where to put the table function(s).

Reply:
Done